src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -418,6 +506,10 @@ void AppleseedRendererPBlockAccessor::Get(
         v.i = static_cast<int>(settings.m_enable_embree);
         break;
 
+      case ParamIdTextureCacheSize:
+        v.i = settings.m_texture_cache_size;
```

Comment:
vs2012 says explicit conversion required: `v.i = static_cast<int>(settings.m_texture_cache_size);`

Reply:
Added the static cast with the last commit